backend/app/api/endpoints/kind/skill_marketplace.py

```
@@ -0,0 +1,366 @@
+
+
+# ==================== Skill Categories ====================
+
+
+@router.get(
+    "/skill-categories",
+    response_model=SkillCategoryListResponse,
+    summary="List all skill categories",
+)
+def list_skill_categories(
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)